packages/@ember/destroyable/type-tests/assert-destroyables-destroyed.test.ts

```diff
@@ -0,0 +1,6 @@
+import { assertDestroyablesDestroyed } from '@ember/destroyable';
+import { expectTypeOf } from 'expect-type';
+
+if (assertDestroyablesDestroyed) {
```

Comment:
Why the conditional? Is it a thing which may not exist at the level of the *import*? 🧐